content/applications/hr/lunch/management.rst

```diff
@@ -0,0 +1,189 @@
+
+.. _lunch/notify:
+
+Notify employees
+----------------
+
+After products are received and the orders are confirmed, the employees must be informed that their
+orders have been delivered and are ready to be picked up.
+
+Unlike sending and confirming orders, notifications must be sent individually and cannot be sent in
+a batch.
+
+To notify the user their product has arrived, click the :guilabel:`✉️ Send Notification` icon at the
+end of each product line. An email is sent to the user informing them their products have been
+delivered.
+
+.. important::
+   It is important for each individual company to configure their day-to-day workflow to include
+   the necessary steps for the *Lunch* app. That includes designating people to be responsible for
+   sending and confirming orders, informing employees when orders have arrived, and determining
+   where orders are stored while waiting to be picked up by employees. These are parts of the
+   *Lunch* app flow that are not configured within Odoo and are determined by each individual
+   company.
+
+.. _lunch/control_vendors:
+
+Control Vendors
+===============
+
+All orders for all vendors, both past and present, can be found in the *Control Vendors* dashboard.
+To access these records, navigate to :menuselection:`Lunch app --> Manager --> Control Vendors`.
+
+All orders appear in a list view, grouped alphabetically by :guilabel:`Vendor`. The list loads with
+all vendors expanded to show all order lines for every vendor by default.
+
+The following information appears in the list:
+
+- :guilabel:`Order Date`: the date the order was placed.
+- :guilabel:`Vendor`: the vendor the product is being ordered from.
+- :guilabel:`Product`: the specific product ordered.
+- :guilabel:`Extras`: any extras selected for the product.
+- :guilabel:`Notes`: any information needed to be sent to the vendor.
+- :guilabel:`User`: the user who ordered the product.
+- :guilabel:`Lunch Location`: where the product is to be delivered.
+- :guilabel:`Price`: the total price for the product, including all extras.
+- :guilabel:`Status`: the current status of the product.
+- :guilabel:`Company`: the company the order was placed under. This only appears in a multi-company
+  database.
+
+Orders can be :ref:`canceled <lunch/cancel>`, :ref:`sent to the vendor <lunch/send-orders>`,
+:ref:`confirmed <lunch/confirm-orders>` upon arrival, and :ref:`employees can be notified
+<lunch/notify>` using the same method as on the :ref:`Today's Orders <lunch/todays-orders>`
+dashboard.
+
+.. image:: management/control.png
+   :align: center
+   :alt: The list of all orders as seen in the Control Vendors dashboard.
+
+.. note::
+   The difference between the :ref:`Today's Orders <lunch/todays-orders>` dashboard and the
+   *Control Vendors* dashboard is that the *Today's Orders* dashboard **only** displays orders for
+   the current day, while the *Control Vendors* dashboard displays **all** orders made in the
+   *Lunch* app.
```

Comment:
```suggestion
   :guilabel:`Control Vendors` dashboard is that the *Today's Orders* dashboard **only** displays orders for
   the current day, while the :guilabel:`Control Vendors` dashboard displays **all** orders made in the
   *Lunch* app.
```
I'm suggesting the use of GUIs for the Control Vendors mentions in this admonition because, at this point in the doc, the reader/user is *on* that page, if they're following along with your instructions. So, since it is technically appearing on the UI at this point, and the Today's Orders dashboard is not, that's why I went with this suggestion.

Reply:
Ahh, OK- becuse I though it is not expliitly on the UI, I don't know where they are in their DB, and there are two different dashboards- they can't be in both places at the same time. I prefer this but I didn't know if it was 'allowed' =)